atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -161,8 +161,8 @@ private void cas(Client client, Long oldVal, long newVal) {
     }
 
     private void logError(String msg, Throwable cause) {
-        log.error(msg, cause);
-        DebugLogger.logger.error(msg, cause);
+        log.error("Error: {}", msg, cause);
+        DebugLogger.logger.error("Error: {}", msg, cause);
```

Comment:
This seems odd to insert an arbitrary `"Error: "` on an error log line. Can we just suppress the FindBugs check for this one?